### ISSUE TITLE
External Timidity++ support?

### ISSUE THREAD

**Issue by sneakernets** [chocolate-doom/chocolate-doom#555]
Would it be possible to add external Timidity/++ support in future?  For example,Randy Heit distributes a custom Timidity++ build and source code that fixes the velocity issues the SDL_Mixer version still exhibits. Unfortunately, the SDL_Mixer timidity also doesn't play fair to BTSX's soundtrack that uses a lot of midi effects that are seldom used.

"Native MIDI" workarounds don't seem to do the trick, either.

**Comment by Xylemon** [comment]
Would love this feature.

**Comment by CraftedPixel** [comment]
Was this in the original doom? Because unfortunately chocolate doom tries to emulate the original as close as possible. I don't think they will add this. Sorry about that.

**Comment by fragglet** [comment]
That is incorrect; this is perfectly within scope for Chocolate Doom. I'm not sure what needs to be done for this - is it a matter of patching SDL_mixer or changing its build setup?
